src/app/models/evt-models.ts

```diff
@@ -331,3 +331,60 @@ export class Word extends GenericElement {
 export class Deletion extends GenericElement {
     rend: string;
 }
+
+export class MsDesc extends GenericElement {
+    id? : string;
+    msIdentifier: MsIdentifier;
+    msContents?: MsContents;
+    physDesc?: PhysDesc;
+    history?: History;
+    msPart?: MsPart;
+}
+
+export class MsIdentifier extends GenericElement {
+    id?: string;
+    settlement?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when settlement is handled
+    repository?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when repository is handled
+    idno?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when idno is handled
+    institution?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when institution is handled
+    altIdentifier?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when altIdentifier is handled
+    msName?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when msName is handled
+    country?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when country is handled
+    region?: Array<ParseResult<GenericElement>>; // TODO: Add specific type when region is handled
```

Comment:
Try to give a definition for each field

Reply:
I personally think it is impossible to handle in detail the definition for every field of every TEI element. At least not in a single PR. We have to stop somewhere. We should probably decide together where it is ok to stop and leave the generic definition.

Reply:
I personally think it is impossible to handle in detail the definition for every field of every TEI element. At least not in a single PR. We have to stop somewhere. We should probably decide together where it is ok to stop and leave the generic definition.

Reply:
for example, `msName` can be described as a `name { type: string, value: string }` instead of an array of `GenericElement`s.
The same for `country { abbr: string, value:string }`. 
This kind of approach is suitable for a lot of these fields, and IMO are bettere definitions than `GenericElement`.

I understand that some of the fields can't be treated like this. But for the rest, I propose to try giving a definition, also because the structure of msdesc seems to hold enough structure.